src/style.css

```diff
@@ -0,0 +1,63 @@
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  --main-color: #00acdc;
+}
+
+.header--nav {
+  box-sizing: border-box;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.logo--link {
+  display: flex;
+}
+
+.nav--list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+}
+
+.nav--item {
+  display: flex;
+  height: 60px;
+}
+
+.nav--link {
+  display: flex;
+  align-items: center;
+  position: relative;
+  line-height: 14px;
+  margin-left: 20px;
+  text-decoration: none;
+  color: #000;
+}
+
```

Comment:
![1](https://user-images.githubusercontent.com/66570142/113003024-01139e80-917b-11eb-91d1-0d150f7b2e8f.png)

Let's remove left margin from this first element. Then all your test will pass (`.nav--link:first-child`)